corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -70,8 +72,12 @@ def _postWithSyncToken(self, filename, token_id):
         process_cases(form)
         return form
 
-    def _postFakeWithSyncToken(self, caseblock, token_id):
-        return post_case_blocks([caseblock], form_extras={"last_sync_token": token_id})
+    def _postFakeWithSyncToken(self, caseblocks, token_id):
+        if not isinstance(caseblocks, list):
+            # can't use list(caseblocks) since that returns children of the node
+            # http://lxml.de/tutorial.html#elements-are-lists
+            caseblocks = [caseblocks]
```

Comment:
is this going to break if you pass in a list? seems like that would result in `[[caseblocks]] `